docker/Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM socrata/java
 ENV SERVER_NAME region-coder
 ENV SERVICE_PORT 2021
```

Comment:
As a followup, you should set SERVICE_2021_NAME to region-coder and SERVICE_2099_NAME to region-coder-jmx in the marathon resource, and ensure they are also in the apps-marathon router config... not sure if the JMX one will work, but would be cool if it did.  Then you could connect to a random instance using the region-coder-jmx apps marathon DNS name.